### PR TITLE
Adding Webhooks selenium tests

### PR DESCRIPTION
**Pull request by kshinde2512**
### Describe your changes :

I worked on the adding selenium tests for Webhooks page

#
### Type of change :

- [x] Improvement
#

### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/open-source-community/developer) document.
- [x] I have performed a self-review of my own. 
- [x] I have tagged my reviewers below.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [x] My changes generate no new warnings.
- [x] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#
### Reviewers
@darth-coder00 @shahsank3t

### CODE REVIEW

File: openmetadata-ui/src/test/java/org/openmetadata/catalog/selenium/objectRepository/Webhooks.java [hunk 1]
```diff
@@ -0,0 +1,68 @@
+package org.openmetadata.catalog.selenium.objectRepository;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+
+public class Webhooks {
+  WebDriver webDriver;
+
+  public Webhooks(WebDriver webDriver) {
```
Comment (mithmatt) [inline openmetadata-ui/src/test/java/org/openmetadata/catalog/selenium/objectRepository/Webhooks.java:9]: You can use lombok RequiredArgsConstructor

File: openmetadata-ui/src/test/java/org/openmetadata/catalog/selenium/pages/Webhooks/WebhooksPageTest.java [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+    webDriver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
+    openWebHookPage();
+    Events.click(webDriver, webhooks.addWebhook());
+    Events.sendKeys(webDriver, webhooks.name(), "test");
+    Events.click(webDriver, webhooks.getDescriptionBox());
+    Events.sendKeys(webDriver, webhooks.getDescriptionBox(), "test");
+    Events.sendKeys(webDriver, webhooks.getEndpoint(), "test.com");
+    Events.click(webDriver, webhooks.getSaveWebhook());
+    WebElement errorMessage = webDriver.findElement(common.errorMessage());
+    if (errorMessage.isDisplayed()) {
```
Comment (mithmatt) [inline openmetadata-ui/src/test/java/org/openmetadata/catalog/selenium/pages/Webhooks/WebhooksPageTest.java:156]: Should the test fail if error message is not displayed?

Comment (kshinde2512) [reply openmetadata-ui/src/test/java/org/openmetadata/catalog/selenium/pages/Webhooks/WebhooksPageTest.java:156]: The test is to check if error message is displayed on sending blank endpoint or blank name. Should I surround the error message with try catch block and throw exception if no such element exists?

Comment (mithmatt) [reply openmetadata-ui/src/test/java/org/openmetadata/catalog/selenium/pages/Webhooks/WebhooksPageTest.java:156]: We expect errorMessage to be displayed. 

So the test would be:
```
if(errorMessae.IsDisplayed()) {
  Assert.assertEquals(errorMessage.getText(), "Webhook event filters are required.");
} else {
  Assert.fail("Expected error message to be displayed")
}
```

Comment (mithmatt) [reply openmetadata-ui/src/test/java/org/openmetadata/catalog/selenium/pages/Webhooks/WebhooksPageTest.java:156]: We can simplify this further...

```
Assert.true(errorMessae.IsDisplayed());
Assert.assertEquals(errorMessage.getText(), "Webhook event filters are required.");
```

Comment (kshinde2512) [reply openmetadata-ui/src/test/java/org/openmetadata/catalog/selenium/pages/Webhooks/WebhooksPageTest.java:156]: Okay,I will make the changes

Comment (darth-coder00) [reply openmetadata-ui/src/test/java/org/openmetadata/catalog/selenium/pages/Webhooks/WebhooksPageTest.java]: @kshinde2512  We should avoid conditions in E2E tests when we are expecting some particular behaviour to occur. This applies for positive as well as negative test cases.

PS: @mithmatt correct me if I have misjudged.

File: openmetadata-ui/src/test/java/org/openmetadata/catalog/selenium/objectRepository/Webhooks.java (hunk 1 above)
Comment (kshinde2512) [reply openmetadata-ui/src/test/java/org/openmetadata/catalog/selenium/objectRepository/Webhooks.java]: Updated lombok @RequiredArgsConstructor

**Review by mithmatt — APPROVED** [review]
Thanks for adding these tests and incorporating review feedback

**Pull request merged**